sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataFeedLiveTests.cs

```diff
@@ -68,462 +68,68 @@ private DateTimeOffset IngestionStartsOnForCustom
         }
 
         [RecordedTest]
-        [TestCase(true)]
-        [TestCase(false)]
-        public async Task CreateAndGetAzureApplicationInsightsDataFeedWithMinimumSetup(bool useTokenCredential)
```

Comment:
We have a lot repeated code in this file, so we merged some of it into new tests.
- All `CreateAndGet<data-feed-kind>WithMinimumSetup` tests (13) have been merged into a single `CreateAndGetWithMinimumSetup` test. There's also a new separate test called `CreateAndGetWithTokenCredential` for AAD testing.
- All `CreateAndGet<data-feed-kind>WithOptionalMembers` tests (13) have been merged into a single `CreateAndGetWithOptionalMembers`.